recipes/jamotools/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "jamotools" %}
+{% set version = "0.1.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "b3dc212ae4fc8dd63ec3ab666c98a6390ac5d28d0adba356ed89360c2d2fb115"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - future
+    - numpy
+    - pip
+    - python
+    - six
+  run:
+    - future
+    - numpy
+    - python
+    - six
+
+test:
+  imports:
+    - jamotools
+    - jamotools.jamo
+    - jamotools.vector
+
+about:
+  home: "https://github.com/HaebinShin/jamotools"
+  license: "GPL"
```

I think it should be:
```suggestion
  license: "GPL-3.0"
```